pdf12.js

```diff
@@ -0,0 +1,133 @@
+const isThereVowels = (nameStudent) => {
```

Comment:
return avgGradesCalc(this.grades) + isThereVowels(this.name)
        ^

ReferenceError: avgGradesCalc is not defined

לא הרצת את התרגיל??